internal/pkg/skuba/deployments/ssh/cni.go

```diff
@@ -33,10 +34,14 @@ func init() {
 }
 
 func cniDeploy(t *Target, data interface{}) error {
```

Comment:
With this refactor I think we can also consider to pass the interface to all states as an argument along with the target and data.

Reply:
Or maybe as a function/attribute on `Target`; maybe it doesn't make sense (depends on how often we use the clientset on the states themselves)